core/server/master/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -171,6 +172,15 @@ void commitBlock(long workerId, long usedBytesOnTier, String tierAlias,
    */
   void commitBlockInUFS(long blockId, long length) throws UnavailableException;
 
+  /**
+   * Marks a block as committed, but without a worker location. This means the block is only in ufs.
+   * Append any created journal entries to the included context.
+   * @param blockId the id of the block to commit
+   * @param length the length of the block
+   * @param context the journal context
+   */
+  void commitBlockInUFS(long blockId, long length, JournalContext context);
```

Comment:
this API only exists in master so I think you can just change it instead of leaving the old one?